common/djangoapps/student/tests/test_reset_password.py

```diff
@@ -113,8 +115,11 @@ def test_reset_password_email(self, send_email):
 
         good_req = self.request_factory.post('/password_reset/', {'email': self.user.email})
         good_req.user = self.user
+        client = Client.objects.create(client_type=constants.CONFIDENTIAL)
+        AccessToken.objects.create(token="test_access_token", client=client, user=self.user)
```

Comment:
Use the factory: `from edx_oauth2_provider.tests.factories import AccessTokenFactory`.